src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -453,7 +453,8 @@ internal static bool IsEdgeSupportedType(Type type)
             {
                 // since HTTP headers are case insensitive, we lower-case the keys
                 // as does Node.js request object
-                headers.Add(header.Key.ToLowerInvariant(), header.Value.First());
+                string values = string.Join(" ", header.Value);
```

Comment:
What will things like accept, accept-encoding, accept-encoding, etc. look like with this logic? Shouldn't those becomma delimited?